src/components/UserInfo/UserInfo.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import UserInfo from ".";
+import { SegmentGroup, Segment } from "semantic-ui-react";
+import "semantic-ui-css/semantic.min.css";
```

Comment:
I'm not sure we need a story for this unless you can imagine us using it elsewhere in the app (not in GameList)